desktop-widgets/downloadfromdivecomputer.cpp

```diff
@@ -639,10 +640,12 @@ void DownloadFromDCWidget::enableBluetoothMode(int state)
 {
 	ui.chooseBluetoothDevice->setEnabled(state == Qt::Checked);
 
-	if (state == Qt::Checked)
-		selectRemoteBluetoothDevice();
-	else
+	if (state == Qt::Checked) {
+		if (ui.device->currentText().isEmpty()) 
+         		selectRemoteBluetoothDevice();
```

Comment:
One more case of spaces - it often pays to enable 'show whitespace' in your editor before submitting code changes.